scripts/quickstart_run.sh

```diff
@@ -77,6 +77,7 @@ fi
 set +e
 echo "Building docker image"
 docker build --pull --rm -f "Dockerfile" --build-arg BINARY=quickstart/service -t cello:latest "."
+docker build --pull --rm -f "Dockerfile.db_migration" -t cello-db-migration:latest "."
```

Comment:
Opted to create a Dockerfile for the image used for the `migrate-db` job to copy the migration files to the image, instead of using the image directly doing some file manipulations to mount a volume.